code/modules/mob/living/simple_animal/hostile/mimic.dm

```diff
@@ -121,8 +121,8 @@ var/global/list/protected_objects = list(/obj/structure/table, /obj/structure/ca
 	var/destroy_objects = 0
 	var/knockdown_people = 0
 
-/mob/living/simple_animal/hostile/mimic/copy/Initialize(loc, var/obj/copy, var/mob/living/creator)
-	. = ..(loc)
+/mob/living/simple_animal/hostile/mimic/copy/Initialize(mapload, obj/copy, mob/living/creator)
```

Comment:
Same here

Reply:
Same question. This fixes a tiny issue, though they have many more.

Reply:
just leave the code for it alone